src/cc/CMakeLists.txt

```diff
@@ -17,6 +17,13 @@ configure_file(libbcc.pc.in ${CMAKE_CURRENT_BINARY_DIR}/libbcc.pc @ONLY)
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fPIC -DBCC_PROG_TAG_DIR='\"${BCC_PROG_TAG_DIR}\"'")
 set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -fPIC")
 
+# set LLVM major version number, only care about version 6 and above
```

Comment:
This will break when 7 comes out? This check seems hacky, why does it need to be 0?

Reply:
LLVM version 7 is a good point. Will use the precise the version number.